src/heap/memory-reducer.cc

```diff
@@ -130,11 +130,19 @@ bool MemoryReducer::WatchdogGC(const State& state, const Event& event) {
          event.time_ms > state.last_gc_time_ms + kWatchdogDelayMs;
 }
 
+inline std::string get_env(const std::string& env_name) {
+  char* ret = getenv(env_name.c_str());
+  return ret ? std::string(ret) : std::string();
+}
+
+inline bool use_membalancer() {
+  return get_env("USE_MEMBALANCER") == "1";
+}
```

Comment:
Let's throw a comment above this saying that this is a stub, we just don't know how to integrate with Chrome's configuration / flags mechanism